theme/DesignTheme/Blocks/_searchServices.scss

```diff
@@ -0,0 +1,80 @@
+.searchServices{
+  width: 100%;
+  min-height: 280px;
+  padding: 10px 50px;
+  @media (max-width: 585px) {
+    padding: 10px 6px;
+  }
+
+  background-color: $secondary;
+
+  .searchContainer {
+    width: 100%;
+    display: flex;
+    padding: 0px 12%;
+    flex-direction: column;
+    justify-content: space-evenly;
+    margin: 56px 0;
```

Comment:
per questi stili ci sono tutte le classi di bootstrap già fatte che si possono inserire nel markup del componente. 
tipo 'd-flex' per fare per fare il componente display flex oppure la classe mt-6 che aggiunge un margin-top circa uguale a 56px ecc. 
Se poi non si riesce ad usarle perchè ci sono già altre classi che sono in mezzo ai piedi allora va bene cosi.